k8s/scripts/provision.sh

```diff
@@ -10,7 +10,7 @@ swapoff -a
 sed -i '/ swap / s/^/#/' /etc/fstab
 
 # Disable spectre and meltdown patches
-sed -i 's/quiet"/quiet spectre_v2=off nopti"/' /etc/default/grub
+sed -i 's/quiet"/quiet spectre_v2=off nopti hugepagesz=2M hugepages=64"/' /etc/default/grub
```

Comment:
@cynepco3hahue can there be kernels where this does not work? Do we need a switch?

Reply:
@rmohr If we talk only about x86_64, it will always work, the problem happens only with 1G hugepages(it requires specific CPU flag, two of our CI hosts does not have this flag)